src/main/java/seedu/address/logic/commands/TrieNode.java

```diff
@@ -0,0 +1,48 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+/**
+ * Class for Trie Node object.
```

Comment:
Give slight better header comment to indicate what it represents.

For example, the `Phone` class.
```
/**
 * Represents a Person's phone number in the address book.
 * Guarantees: immutable; is valid as declared in {@link #isValidPhone(String)}
 */